src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -80,18 +86,24 @@ def send_response_callback(response):
 
 class HealthServicer(_health_pb2_grpc.HealthServicer):
     """Servicer handling RPCs for service statuses."""
+    _lock: threading.RLock
+    _server_status: Mapping[str, Any]
+    _send_response_callbacks: Mapping[str, Callable[[Any], None]]
```

Comment:
I think this is `Mapping[str, Set[Callable[[health_pb2.HealthCheckResponse], None]]]`